scripts/deploy.sh

```diff
@@ -30,21 +30,25 @@ function initialize {
 
 function upload_assets {
   git clone --quiet --depth 1 git@github.com:${SITE_REPO_SLUG}.git "$SITE_HOME"
-  mv -f "$TLDR_ARCHIVE" "$SITE_HOME/assets/"
+  [[ -f "$TLDR_ARCHIVE" ]] && mv -f "$TLDR_ARCHIVE" "$SITE_HOME/assets/"
   find "$TLDR_LANG_ARCHIVES_DIRECTORY" -maxdepth 1 -name "*.zip" -exec mv -f {} "$SITE_HOME/assets/" \;
   rm -rf "$TLDR_LANG_ARCHIVES_DIRECTORY"
-  cp -f "$TLDRHOME/index.json" "$SITE_HOME/assets/"
+  [[ -f "$TLDRHOME/index.json" ]] && cp -f "$TLDRHOME/index.json" "$SITE_HOME/assets/"
   find "$TLDR_PDF_FILES_DIRECTORY" -maxdepth 1 -name "*.pdf" -exec mv -f {} "$SITE_HOME/assets/" \;
   rm -rf "$TLDR_PDF_FILES_DIRECTORY"
 
   cd "$SITE_HOME/assets"
   sha256sum -- index.json *.zip > tldr.sha256sums
 
-  git add -A
-  git commit -m "[GitHub Actions] uploaded assets after commit tldr-pages/tldr@${GITHUB_SHA}"
-  git push -q
-
-  echo "Assets (pages archive, index) deployed to static site."
+  # Check if there are changes before committing and pushing.
+  if git diff --quiet; then
+    echo "No changes to deploy."
+  else
+    git add -A
+    git commit -m "[GitHub Actions] uploaded assets after commit tldr-pages/tldr@${GITHUB_SHA}"
+    git push -q
+    echo "Assets (pages archive, index) deployed to the static site."
+  fi
```

Comment:
```suggestion
  # Check if there are changes before committing and pushing.
  if git diff --quiet; then
    echo "No changes to deploy."
    return
  fi

  git add -A
  git commit -m "[GitHub Actions] uploaded assets after commit tldr-pages/tldr@${GITHUB_SHA}"
  git push -q
  echo "Assets (pages archive, index) deployed to the static site."
```

Reply:
Yeah, this looks much better. Minor nitpick in the commit title along with archive and index can you add checksum too?

Reply:
> Yeah, this looks much better. Minor nitpick in the commit title along with archive and index can you add checksum too?

What do you mean? Add "don't compute checkums" to the commit message? You can change the commit message before merging if you want to.

Reply:
> > Yeah, this looks much better. Minor nitpick in the commit title along with archive and index can you add checksum too?
> 
> What do you mean? Add "don't compute checkums" to the commit message? You can change the commit message before merging if you want to.

Sorry, I confused the commit message and log, I meant updating the log's message to `echo "Assets (pages archive, index and checksum) deployed to the static site.`